deltacat/compute/compactor_v2/utils/exception_handler.py

```diff
@@ -0,0 +1,67 @@
+from typing import Optional
+from ray.exceptions import RayError, RuntimeEnvSetupError
+from pyarrow.lib import ArrowException, ArrowInvalid, ArrowCapacityError
+from deltacat.exceptions import (
+    DeltaCatError,
+    DependencyPyarrowError,
+    DependencyRayRuntimeSetupError,
+    DependencyPyarrowInvalidError,
+    DependencyPyarrowCapacityError,
+)
+
+
+def parametrized(dec):
+    def layer(*args, **kwargs):
+        def repl(f):
+            return dec(f, *args, **kwargs)
+
+        return repl
+
+    return layer
+
+
+@parametrized
+def handle_exception(func, task_id):
```

Comment:
can we get `task_id` inside this function?

Reply:
Any reason for not resolving this?

Reply:
Did't get your original comment. Are you asking if we are able to pass `task_id`?

Reply:
This `handle_exception` is used as a function annotation to all Ray remote tasks. Not sure how to fetch it inside?

Reply:
Just call `get_current_ray_task_id` within this function :)

Reply:
Oh I see what you mean, will update in next revision.

Reply:
Any update on this PR?